redaxo/src/core/lib/var_dumper.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Symfony\Component\VarDumper\Cloner\VarCloner;
+use Symfony\Component\VarDumper\Dumper\CliDumper;
+use Symfony\Component\VarDumper\Dumper\DataDumperInterface;
+use Symfony\Component\VarDumper\Dumper\HtmlDumper;
+use Symfony\Component\VarDumper\VarDumper;
+
+/**
+ * @package redaxo\core
+ */
+abstract class rex_var_dumper
+{
+    /** @var VarCloner */
+    private static $cloner;
+
+    /** @var DataDumperInterface */
+    private static $dumper;
+
+    public static function register()
+    {
+        VarDumper::setHandler(function ($var) {
+            if (rex::isDebugMode() || ($user = rex_backend_login::createUser()) && $user->isAdmin()) {
+                VarDumper::setHandler('self::dump');
+                self::dump($var);
+
+                return;
+            }
+
+            // register noop handler for non-admins (if not in debug mode)
+            VarDumper::setHandler(function ($var) {
```

Comment:
brauchen wir das wirklich? würde es nicht reichen einfach nur zu tun.. ? wir setzen ja bereits einen handler in zeile 22?